scripts/common.psm1

```diff
@@ -143,9 +143,10 @@ function CommitUpdatedVersions([hashtable]$updatedVars, [xml]$dependencies, [str
         $remote = "origin"
         $baseBranch = "dev"
 
+        Invoke-Block { & git stash }
```

Comment:
What's this about?

Reply:
We've already made changes to build/dependencies.props, if we don't stash our changes before we checkout checkout will fail.

Reply:
Is there any reason we cannot but the branch checkout to the top of the script? stash pop is prone to merge conflicts.